src/index.hot.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppContainer as Hot } from 'react-hot-loader';
+
+import RootRender from '../src';
+
+const render = () => {
```

Comment:
nope.  i found the bug QQ.

I gave the randered react-dom to be a react-dom QQ.